{"filename": "python-retrying-1.3.3-16-any.pkg.tar.zst", "name": "python-retrying", "base": "python-retrying", "version": "1.3.3-16", "desc": "A general-purpose retrying library", "csize": "12872", "isize": "42030", "md5sum": "b40e7134eaa8b9a2d9e3216802f23ced", "sha256sum": "a99d16af5086b773e2e101a8b6a9fa20249be01a4e966ef2056d38393fed4fcf", "pgpsig": "iQIzBAABCAAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmQvAv4ACgkQeGxj8zDXy5JhyQ/+LW3HZuL8KCZlBEbrTIBqN6o/04WPFPJr4dB4PDDGryBWEuoRsuCpjvGLXq5MJNgZfuOkbHK8S0yozhyHFpETxCUdYrZOjNoiEHG+z9IQhcmrj7WHsc8HG2c8ELNHTdVqHQxa1l5as5zvmas8Fw9JvsCTNQ054e/WF3Gja2CcWsaZxBh2sXgeoDRtcwLeY3EQK1/PCXwZ3zh04LOYeBDHgwRKzdNj1uNEA+eFhw+5sAPa9dxDlNn00ttKNlnlLDi2kXVaJKVb00gmnONAHd7pnaX7oPZj/sHWSPZcbUMZcBLf5un0qKpemEbUzsv+gitCsfyDqiGAv4IpEkXzUn72M8CUEycoW7lMbtdbrSJifCKWS2N8GVyG/5JJlrzztsG9z8jer0rG2KxLcA6sn+gxTGd2z8e8Re9xrlcLJTpeRw6hIsO9g1gF/3+NewjaZVpUp1aqJXMyb7jgIiGKc5+n4Q/WAv/iXqz4iuG25An1PEbPHJsqE22p/+gH4KcVYeHd2gnYihe7MlAvSRuGSfiPQDJCohWrNK5ksU6TLcCGKLBUkGl8rTHtHWmSrvZA4iE6ux8GRyjNL1kJOemP5zX1YpmL2OG73KZmCIkOUxDRpKWaIBrE2GLbydSH1lfrCw0f0xZ3BdqNKC8I/xIhYwSXI4TbVCpbbnjDls7GPo50nTA=", "url": "https://github.com/rholder/retrying", "license": "Apache", "arch": "any", "builddate": "1680802542", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["python-six"], "makedepends": ["python-setuptools"], "tokens": ["python-retrying", "python-retrying_x86_64", "python-retrying_unstable", "python-retrying_x86_64_unstable"]}